### PR TITLE
errors tutorial iteration 

### PR DESCRIPTION
**Pull request by akristen**
PR for updated errors tutorial

### CODE REVIEW

File: src/content/docs/apm/agents/manage-apm-agents/agent-data/manage-errors-apm-collect-ignore-or-mark-expected.mdx [hunk 1]
```diff
@@ -13,11 +13,15 @@ redirects:
   - /docs/agents/manage-apm-agents/agent-data/manage-errors-apm-collect-ignore-mark-expected
 ---
 
-[APM](/docs/apm) agents automatically report error data for [supported frameworks](/docs/agents/manage-apm-agents/installation/compatibility-requirements). To optimize error reporting and alerting, you can further manage errors in order to:
+APM agents automatically report error data for [supported frameworks](/docs/agents/manage-apm-agents/installation/compatibility-requirements). To optimize error reporting and alerting, you can further manage errors in order to:
 
-* [Catch errors](#error-collection) that we don't instrument by default.
-* [Ignore errors](#ignore) that you don't want reported at all.
-* Filter out noise from [expected errors](#expected) so you can focus on the errors that are affecting performance. (Java, Ruby, Node, Python, and .NET agents only)
+* Catch errors that we don't instrument by default.
```
Comment (homelessbirds) [inline src/content/docs/apm/agents/manage-apm-agents/agent-data/manage-errors-apm-collect-ignore-or-mark-expected.mdx:18]: Why did you take out the additional links @akristen?

Comment (akristen) [reply src/content/docs/apm/agents/manage-apm-agents/agent-data/manage-errors-apm-collect-ignore-or-mark-expected.mdx:18]: They were pretty noisy at the start of the doc, and people can use the quick nav on the right to jump to different sections of the doc. I think including a bunch of links at the start of the doc can create a lot of link fatigue and distract from just tackling the content yourself.

**Review by homelessbirds — APPROVED** [review]
Approved pending a working build.

**Comment by homelessbirds** [comment]
Approved, saw the working build link.

**Comment by akristen** [comment]
Merging per working build from a manual trigger: https://build-bad574e1-1d62-4185-8682-0239d0e8957a.gatsbyjs.io/docs/tutorial-error-tracking/respond-outages/

**Pull request merged**